package/kernel/linux/modules/usb.mk

```
@@ -519,6 +519,23 @@ endef
 $(eval $(call KernelPackage,usb-dwc3))
 
 
+define KernelPackage/usb-dwc3-octeon
+  TITLE:=DWC3 Cavium Octeon USB driver
+  DEPENDS:=@(TARGET_octeon) +kmod-usb-dwc3
```

Please do it like this:
```suggestion
  DEPENDS:=@TARGET_octeon +kmod-usb-dwc3
```

---

done